src/index.html

```diff
@@ -724,7 +724,7 @@ <h5>Submission validation processes</h5>
                                             <p><span style="font-size:14px;font-weight:bold;">VCF Specification:</span></p>
                                             <p>
                                                 All VCF files submitted to EVA are validated for adherence to the <a href="https://samtools.github.io/hts-specs/" target="_blank">standard format</a> using the <a href="https://github.com/EBIvariation/vcf-validator" target="_blank">EVA VCF validation suite</a> which includes all the checks from the vcftools suite, and some more that involve lexical, syntactic and semantic analysis of the VCF input.
-                                                The EVA VCF validation suite also includes a debugging tool to automatically correct many of the common errors found in files. In order to improve processing time, submitters are encouraged to pre-validate VCF files prior to submission.
+                                                The EVA VCF validation suite also includes a debugging tool to automatically correct many of the common errors found in files and an assembly checker tool to verify variants match a reference sequence. In order to improve processing time, submitters are encouraged to pre-validate VCF files prior to submission.

```

Comment:
@cyenyxe @jmmut @BaronKoy  I have often seen this phrasing in our documentation and we don't bat an eye because it makes sense to us. For an outside submitter, wouldn't it be more accurate to say "to validate the reference alleles in the VCF files against the reference sequence". "Variants matching a reference sequence" sounds contradictory by definition but I am not sure if this is widespread in the bioinformatics literature. Just as a random check, I looked up how NCBI describes such validation and this is how it goes "A variant defined as a sequence change relative to a reference sequence is analyzed to verify that the reference allele occurs at the position reported on the reference sequence." See https://www.ncbi.nlm.nih.gov/clinvar/docs/validation/